scripts/generate_records/generate_db_records.js

```diff
@@ -0,0 +1,439 @@
+// @ts-check
+/* eslint-disable node/no-extraneous-require */
+/* eslint-disable no-await-in-loop */
+const pMap = require('p-map');
+const minimist = require('minimist');
+const cryptoRandomString = require('crypto-random-string');
+const cliProgress = require('cli-progress');
+
+const { randomInt } = require('crypto');
+const {
+  GranulePgModel,
+  CollectionPgModel,
+  ProviderPgModel,
+  ExecutionPgModel,
+  GranulesExecutionsPgModel,
+  FilePgModel,
+  getKnexClient,
+} = require('@cumulus/db');
+const {
+  loadCollection,
+  loadExecutions,
+  loadProvider,
+  loadFiles,
+  loadGranulesExecutions,
+  loadGranules,
+} = require('./db_record_loaders');
+
+/**
+ * @typedef {import('@cumulus/db').PostgresFile} PostgresFile
+ * @typedef {import('@cumulus/db').PostgresGranule} PostgresGranule
+ * @typedef {import('@cumulus/db').PostgresCollection} PostgresCollection
+ * @typedef {import('knex').Knex} Knex
+ * @typedef {{
+ *   geModel: GranulesExecutionsPgModel,
+ *   executionModel: ExecutionPgModel,
+ *   granuleModel: GranulePgModel,
+ *   fileModel: FilePgModel
+ * }} ModelSet
+ * @typedef {{
+ *   name: string,
+ *   version: string,
+ * }} CollectionDetails
+ */
+
+process.env.DISABLE_PG_SSL = 'true';
+/**
+ * yield series of collection details
+ *
+ * @param {number} total - number of collections
+ * @param {boolean} repeatable - use consistent names versus pseudorandom
+ * @yields {CollectionDetails}
+ */
+function* yieldCollectionDetails(total, repeatable = true) {
+  for (let i = 0; i < total; i += 1) {
+    let suffix;
+    if (repeatable) {
+      suffix = `_test_${i.toString().padStart(3, '0')}`;
+    } else {
+      suffix = `_${cryptoRandomString({ length: 5 }).toUpperCase()}`;
+    }
+    yield {
```

Comment:
I think this yield block could use a bit of explanation similar to the other generator function below.

Reply:
https://github.com/nasa/cumulus/pull/3679/commits/3ba1b2e8c2befb1eb88aa179b05a79acfe6b9ebd
yea this is a leftover that's gone now in shifting to fake collections